### PR TITLE
Add flow parameter for tracking yesterday's flow.

### PR DESCRIPTION
**Pull request by jetuk**
- Includes a test and test model.

### CODE REVIEW

File: pywr/parameters/_parameters.pyx [hunk 1]
```diff
@@ -1377,6 +1377,56 @@ cdef class DeficitParameter(Parameter):
...
+            num_comb = 1
+        self.__next_values = np.empty([num_comb], np.float64)
+
+    cpdef reset(self):
+        self.__values[...] = 0.0
+
+    cdef calc_values(self, Timestep timestep):
+        cdef int i
+        for i in range(self.__values.shape[0]):
+            print(i, self.__next_values[i], timestep)
```
Comment (snorfalorpagus) [inline pywr/parameters/_parameters.pyx]: remove debug

File: pywr/parameters/_parameters.pyx [hunk 2]
```diff
@@ -1377,6 +1377,56 @@ cdef class DeficitParameter(Parameter):
 DeficitParameter.register()
 
 
+cdef class FlowParameter(Parameter):
+    """Parameter that provides the flow from a node from the previous time-step.
```
Comment (snorfalorpagus) [inline pywr/parameters/_parameters.pyx]: What is the value of this parameter on the initial timestep? Does it need an initial value?

Comment (jetuk) [reply pywr/parameters/_parameters.pyx]: Good point. It is currently zero and not configurable.

File: pywr/parameters/_parameters.pyx (hunk 1 above)
Comment (jetuk) [reply pywr/parameters/_parameters.pyx]: Done

File: pywr/parameters/_parameters.pyx [hunk 3]
```diff
@@ -1377,6 +1377,56 @@ cdef class DeficitParameter(Parameter):
...
+
+    Parameters
+    ----------
+    model : pywr.model.Model
+    node : Node
+      The node that will have its flow tracked
+
```
Comment (snorfalorpagus) [inline pywr/parameters/_parameters.pyx]: Need to add `initial_value` to docstring.

**Pull request merged**